manifests/rootsync-crd.yaml

```diff
@@ -444,6 +444,33 @@ spec:
                           x-kubernetes-int-or-string: true
                       type: object
                     type: array
+                  roleRefs:
+                    description: roleRefs is a list of Roles or ClusterRoles to create
+                      bindings. If unset, a binding to cluster-admin will be created.
```

Comment:
Presumably then the cluster-admin binding will also be removed if the user adds a roleRefs through an update?

Reply:
Yes that is correct, if a user adds roleRefs then the `cluster-admin` default will be removed. Conversely if a user removes all roleRefs then the custom bindings will be deleted and the `cluster-admin` default will be re-created.